### PR TITLE
Adding Left on Read Premium

### PR DESCRIPTION
**Pull request by Teddarific**

### CODE REVIEW

File: server/src/controllers/PremiumController.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    }
+    return res.status(StatusCodes.BAD_REQUEST).send(`Webhook Error: ${err}`)
+  }
+
+  if (event.type === 'payment_intent.succeeded') {
+    const stripeObject: Stripe.PaymentIntent = event.data
+      .object as Stripe.PaymentIntent
+
+    // const customerEmail = stripeObject.charges?.data[0].billing_details.email
+    const customerEmail = 'teddyni19@gmail.com'
```
Comment (Teddarific) [inline server/src/controllers/PremiumController.ts:54]: unhardcode this

File: app/src/components/Dashboard/Navbar.tsx [hunk 2]
```diff
@@ -147,6 +148,39 @@ export function Navbar({
           <Menu>
             <MenuButton as={IconButton} icon={<SettingsIcon />} size="md" />
             <MenuList>
+              {!isPremium && (
+                <MenuItem
+                  onClick={() => {
+                    onRefresh();
```
Comment (alexdanilowicz) [inline app/src/components/Dashboard/Navbar.tsx:154]: This is wrong onClick I think. Do you want? Or open the stripe page?
```suggestion
      ipcRenderer.invoke('active-license');
```

File: app/src/components/Premium/PremiumModal.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            </>
+          ) : (
+            <>
+              <ModalHeader>Unlock Premium</ModalHeader>
+              <ModalCloseButton />
+
+              <ModalBody>
+                <div>
+                  <Text fontWeight="medium" style={{ marginBottom: 6 }}>
+                    Your License Key
```
Comment (alexdanilowicz) [inline app/src/components/Premium/PremiumModal.tsx:154]: I think slightly confusing language. Maybe
```suggestion
                    Input Your License Key
```

File: app/src/components/Premium/PremiumModal.tsx [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+                  </Text>
+                  <Input
+                    placeholder="LOR-365f9d34-6d39-4868-ae71"
```
Comment (alexdanilowicz) [inline app/src/components/Premium/PremiumModal.tsx:157]:
```suggestion
                    placeholder="Check your email!"
```

File: app/src/components/Dashboard/Navbar.tsx (hunk 2 above)
Comment (Teddarific) [reply app/src/components/Dashboard/Navbar.tsx:154]: Oh oops good catch, this needs to open modal

**Pull request merged**